server/backend/api/config/config.py

```diff
@@ -167,6 +171,11 @@ def get_idim_proxy_api_key():
     return idim_proxy_api_key
 
 
+def get_gc_notify_email_api_key():
```

Comment:
This is the only change in this file, the rest are just done by the auto formatter